src/KingSystem/System/PlayReportMgr.cpp

```diff
@@ -13,6 +13,40 @@ PlayReportMgr::~PlayReportMgr() {
         delete mReporter;
 }
 
+void PlayReportMgr::calc() {
+    if (!_30 && mReporter)
+        mReporter->updateTimers();
+}
+
+void PlayReportMgr::reportDebug(const sead::SafeString& message, const sead::SafeString& data) {
+    // Stubbed in release builds
+}
+
+int PlayReportMgr::auto0() const {
+    return 1;
+}
+
+PlayerTrackReporter* PlayReportMgr::auto2() const {
+    if (mReporter)
```

Comment:
I think this would be more natural:

```cpp
if (!mReporter)
    return nullptr;
return mReporter->getPlayerTrackReporter();
```
(or even a ternary)